.travis.yml

```diff
@@ -15,15 +15,15 @@ matrix:
     - python: '2.7'
       env:
         - TOXENV=du13
-        - PYTEST_ADDOPTS = --cov sphinx --cov-append --cov-config setup.cfg
+        - PYTEST_ADDOPTS="--cov ./ --cov-append --cov-config setup.cfg"
     - python: '3.4'
       env: TOXENV=py34
     - python: '3.5'
       env: TOXENV=py35
     - python: '3.6'
       env:
         - TOXENV=py36
-        - PYTEST_ADDOPTS = --cov sphinx --cov-append --cov-config setup.cfg
+        - PYTEST_ADDOPTS="--cov ./ --cov-append --cov-config setup.cfg"
```

Comment:
The example of codecov says `./` is better.
https://github.com/codecov/example-python#pytest